docker/Dockerfile_base

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:22.04
 RUN groupadd --gid 1000 node \
   && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install git python3 python2 build-essential avahi-daemon avahi-discover avahi-utils libnss-mdns mdns-scan libavahi-compat-libdnssd-dev sysstat procps nano curl libcap2-bin sudo \
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install git python3 python2 build-essential avahi-daemon avahi-discover avahi-utils libnss-mdns mdns-scan libavahi-compat-libdnssd-dev sysstat procps nano curl libcap2-bin sudo dbus bluez\
```

Comment:
We had already dbus previously, but does it need to be installed explicitly now for bluez support?

Reply:
I think it is not needed, but I added there just to ensure and remind that dbus is needed. If someone use some other base, then critical dependenses might be good to be listed here. Can be removed also.